Deferred/LockProtected.swift

```diff
@@ -6,8 +6,6 @@
 //  Copyright © 2014-2015 Big Nerd Ranch. Licensed under MIT.
 //
 
-import Foundation
-
```

Comment:
I don't see this file being used by anything other than tests.  If it isn't needed, I think it should be removed from the repo.